pkg/webui/console/lib/utils.js

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2022 The Things Network Foundation, The Things Industries B.V.
```

Comment:
We have been using only atomic utility modules so far. If we wanted to use a general `utils.js` then we'd have to refactor all our existing utils into that to be consistent. To avoid that I suggest to use something like `lib/duration-utils.js` here.

See my comment below though, we might not need this module at all.